### DISCUSSION TITLE
User based recommandations

### DISCUSSION THREAD

**Discussion by PierreDurrr** [lostb1t/replex#136] [General]
Based on this [https://github.com/lostb1t/replex/issues/132#issuecomment-2227281219](https://github.com/lostb1t/replex/issues/132#issuecomment-2227281219) and now replex can Filter hub with label permissions, you can create recommendations of movies/shows for you and your users. 
Here is how i do it personally:

* using tautulli
* created a trakt account for each of my user
* linked tautulli user history to trakt.tv user account (using [Tautulli2Trakt](https://github.com/PierreDurrr/tautulli2trakt/tree/master) as i was already using tautulli, can probably be done using PlexTraktSync) 
* (If not using tautulli, i think you can do this using [GoPlaxt (online)](https://plaxt.astandke.com/) or [selfhosted (check their github)](https://github.com/xanderstrike/goplaxt) and Plex webhooks (webhooks requires Plex Pass)
* use couchmoney.tv to generate some trakt lists of recommended content, based on watch history
* use radarr/sonarr to get content if needed
* use kometa (formerly plex-meta-manager) to generate collection, apply label to them and pin to users home.
* set label restrictions for my users in plex settings (can probably be done automagically using JBOPS scripts)

And for an example of collection, there you go :

```yml
collections:

  ✨ Nouveau ! ✨ Élise, voici les films recommandés pour toi:
    sort_title: "zzz_✨ Nouveau ! ✨ Élise, voici les films recommandés pour toi"
    summary: "Ces recommandations sont basées sur ton historique de lecture, et sur les notes que tu as données aux films que tu as regardés."
    trakt_list: https://trakt.tv/users/trakt_elise/lists/movies-based-on-recently-watched-couchmoney-tv?sort=rank,asc
    limit: 40 
#    url_poster: poster-a-ajouter
    collection_order: custom
    collection_mode: show_items
    visible_shared: true
    visible_library: true
    sync_mode: sync
    label: 
      - trakt_elise
      - REPLEXHERO

**Comment by stevezau** [comment]
@PierreDurrr do run [Tautulli2Trakt](https://github.com/PierreDurrr/tautulli2trakt/tree/master) per user? I have several users so trying to work out the best way to manage this with min overheads. 

thanks!

**Reply by stevezau** [reply]
guess i could do https://github.com/Taxel/PlexTraktSync/discussions/997

**Reply by PierreDurrr** [reply]
> @PierreDurrr do run [Tautulli2Trakt](https://github.com/PierreDurrr/tautulli2trakt/tree/master) per user? I have several users so trying to work out the best way to manage this with min overheads.
> 
> thanks!

Yes, i got one copy of Tautulli2Trakt per user.
Maybe can be done with PlexTraktSync aswell but last time i tried, it messed both mine and my users history (probably misconfigured...)

**Reply by stevezau** [reply]
can you share your config? I looked at [Tautulli2Trakt](https://github.com/PierreDurrr/tautulli2trakt/tree/master) code but couldn't see how it is filtering by user?

**Reply by PierreDurrr** [reply]
It is not filtering by user. What i've done is making a copy of the script for each of my user, renaming for ease of use, and configuring. Then, i've made one entry per user in Tautulli Notification settings.
